### ISSUE TITLE
options_map misses entries from multiple import directives

### ISSUE THREAD

**Issue by blais** [beancount/beancount#91]
**[Original report](https://bitbucket.org/blais/beancount/issue/91) by Dominik Aumayr (GitHub: [aumayr](https://github.com/aumayr)).**

----------------------------------------

*This was first reported by [yegle](https://github.com/yegle) [here](https://github.com/aumayr/beancount-web/issues/9).*

It looks like `beancount.loader.load_file` does not return a correct `options_map` when using `include` directives: The keys `operating_currency` and `commodities` are missing or reporting the wrong value.

Here's a minimal test case that demonstrate the problem.

invest.bean
```
include "accounts.bean"
include "a.bean"
include "b.bean"
```

accounts.bean
```
2015-12-09 open Assets:AccountA
2015-12-09 open Assets:AccountB
```

a.bean
```
include "accounts.bean"

2015-12-09 *
    Assets:AccountA -1 USD
    Assets:AccountB 1 USD
```

b.bean
```
include "accounts.bean"

2015-12-10 *
    Assets:AccountA -1 USD
    Assets:AccountB 1 USD
```

`beancount.loader.load_files` returns an `options_map` without the key `commodities`. When defining `option "operating_currency" "USD"` in one of the included files this key is also missing from `options_map`.

Use `bean-report -f beancount invest.bean print > /tmp/output.beancount` to `concatenate` everything into a single file, and `beancount.loader.load_files` returns a correct `options_map`.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

This is a known issue. Only the top-level's options_map is filled in. I haven't defined a sensible way to aggregate multiple options_maps together, which is required to do this correctly. Another option would be for me to recycle the options_map across multiple files. This will require more thought and possibly some wide-ranging changes, not a small task I can address right now.

I'd rather pick and choose to address those specific fields you need in the meantime.
Is the "commodities" the only field you need?

**Comment by blais** [comment]
**Original comment by Dominik Aumayr (GitHub: [aumayr](https://github.com/aumayr)).**

----------------------------------------

Only the two fields `commodities` and `operating_currency` are the ones I need for `beancount-web` to work properly.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #91: Aggregated some of the attributes of the options_map across included files.

**Comment by blais** [comment]
**Original comment by Dominik Aumayr (GitHub: [aumayr](https://github.com/aumayr)).**

----------------------------------------

Works now, thanks!
